src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms;
```

Comment:
using `all` is a bit of bad practice. it'd be better to explicitly specify properties here